src/retabled/core.cljc

```diff
@@ -86,12 +93,14 @@
         [:table.table (assoc {:id table-id} :style (when (:table-scroll-bar controls)
                         style-table))
          [internal/generate-theads {:controls controls
-                           :paging-controls  paging-controls
-                           :SORT  SORT
-                           :FILTER  FILTER
-                           :table-id  table-id}]
+                                    :paging-controls  paging-controls
+                                    :SORT  SORT
+                                    :FILTER  FILTER
+                                    :table-id  table-id
+                                    :columns columns}]
```

Comment:
Make this a namespaced kw to indicate that it is internal-only: `:internal/columns`